### DISCUSSION TITLE
Separator electronic conductivity

### DISCUSSION THREAD

**Discussion by kabravenkatesh** [pybamm-team/PyBaMM#3201] [Q&A]
Hello all,

I am wanting to know if I can set a certain value of the separator electronic conductivity in the model. Also how do I change the boundary conditions for the electrode charge conservation equation. Can you please point me to any tutorial if present. 

Thanks a lot!

**Comment by valentinsulzer** [comment]
The separator is an insulator, it doesn't have an electronic conductivity.
Which boundary conditions to you want to change and from what to what?

**Reply by kabravenkatesh** [reply]
So usually the anode-separator and cathode-separator interfaces are assumed to have zero electronic flux i.e. neumann boundary condition for solid phase potential. 

I would like to solve the charge conservation equation in the separator as well with finite separator electronic conductivity. In a sense consider separator as an electrode with zero source term and I would like to apply BC for the solid phase potential to have a some gradient in the potential at the electrode separator interfaces. (unlike zero earlier)

**Reply by brosaplanella** [reply]
At the moment we assume that separators are perfect insulators so we only write electrolyte equations. To include this in PyBaMM you would have to add a new separator submodel, even though it could probably leverage the existing electrode models. If you just want to write a simpler model to test some ideas, you can write a model from scratch, as explained in these notebooks:

* https://docs.pybamm.org/en/latest/source/examples/notebooks/creating_models/5-half-cell-model.html
* https://docs.pybamm.org/en/latest/source/examples/notebooks/creating_models/6-a-simple-SEI-model.html
